lib/supported-selector.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+// Create a shortcut for the CSSRule interface
+var CSSRule = window.CSSRule
+
+// Define prefixes as they are used in this context
+var prefixes = ['MOZ_', 'WEBKIT_', 'O_', 'MS_', '']
+
+var cache = {}
+
+/**
+ * Test if a selector is supported, returns supported selector with vendor
+ * prefix if required.
+ *
+ * Returns selector if at-rule is not present.
+ * Returns at-rule if present and feature-detected. Rule is vendor prefixed if required.
+ * Returns `false` if at-rule is present and not supported.
+ *
+ * @param {String} selector
+ * @return {String|Boolean}
+ * @api public
+ */
+module.exports = function (selector) {
+    if (selector[0] !== '@') {
+        // No at-rule present to feature detect.
+        return selector
+    }
+
+    var selectorTokens = selector.split(' ')
+    var atRule = selectorTokens[0].substr(1)
+    var remainderTokens = selectorTokens.slice(1)
+    var supportedAtRule = cache[atRule] || getSupportedAtRule(atRule)
+
+    return supportedAtRule && [supportedAtRule]
```

Comment:
Ah yeah supportedAtRule can only be a string or false here so when its falsy it will just return false.

can we change the syntax to?

```
if (supportedAtRule) {
    return supportedAtRule + ' ' + remainderTokens.join(' ')
}

return false
```
or something similar